### PR TITLE
Add io.github.endless_sky.endless_sky

### PR DESCRIPTION
**Pull request by corecontingency**
See [here](https://github.com/flathub/io.github.EndlessSky.endless-sky/issues/2).

Currently, because Endless Sky has a different AppStream ID in different distribution channels, it confusingly shows up as different apps in clients that can display content from multiple sources, such as GNOME Software and KDE Discover.

By renaming the Flathub AppStream ID to the correct one used upstream, then Discover and GNOME Software (and similar apps) can perform de-duplication correctly, marking both as the same app and offering the user the choice of which version they want to install.

I will mark the old repository as EOL once this is merged.

### CODE REVIEW

File: io.github.endless_sky.endless_sky.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          "sha256": "4cea417fdd7499a36f407923d03b4b7000b0f9e8fd7b31b316b9ce7eba9143a5"
+        }
+      ]
+    },
+    {
+      "name": "endless-sky",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ln -s /bin/python3 /app/bin/python",
+        "CPPPATH=/app/include scons PREFIX=/app install -j$(nproc)",
```
Comment (bilelmoussaoui) [inline io.github.endless_sky.endless_sky.json:39]:
```suggestion
        "CPPPATH=/app/include scons PREFIX=/app install -j$(FLATPAK_BUILDER_N_JOBS)",
```

File: io.github.endless_sky.endless_sky.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          "url": "https://downloads.sourceforge.net/project/scons/scons/3.1.1/scons-3.1.1.tar.gz",
+          "sha256": "4cea417fdd7499a36f407923d03b4b7000b0f9e8fd7b31b316b9ce7eba9143a5"
+        }
+      ]
+    },
+    {
+      "name": "endless-sky",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ln -s /bin/python3 /app/bin/python",
```
Comment (bilelmoussaoui) [inline io.github.endless_sky.endless_sky.json:38]: Just call scons with `/app/bin/python3 scons` instead of this

**Comment by bilelmoussaoui** [comment]
bot, build io.github.endless_sky.endless_sky

**Comment by corecontingency** [comment]
Thanks for the changes! Not sure why the build is failing for ARM, I didn't change the manifest and it built fine before.

Let's try it again.

bot, build

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

**Comment by corecontingency** [comment]
@bilelmoussaoui 

> Just call scons with /app/bin/python3 scons instead of this

Unfortunately, this doesn't seem to work. 
`/bin/sh: /app/bin/python3: No such file or directory`
Tried `/bin/python3 scons` also.
`/bin/python3: can't open file 'scons': [Errno 2] No such file or directory`

Also, replacing `-j$(nproc)",` with `-j$(FLATPAK_BUILDER_N_JOBS)",` doesn't seem to work either.
`/bin/sh: FLATPAK_BUILDER_N_JOBS: command not found`

I reverted the commit.

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui
> 
> > Just call scons with /app/bin/python3 scons instead of this
> 
> Unfortunately, this doesn't seem to work.
> `/bin/sh: /app/bin/python3: No such file or directory`
> Tried `/bin/python3 scons` also.
> `/bin/python3: can't open file 'scons': [Errno 2] No such file or directory`
> 
> Also, replacing `-j$(nproc)",` with `-j$(FLATPAK_BUILDER_N_JOBS)",` doesn't seem to work either.
> `/bin/sh: FLATPAK_BUILDER_N_JOBS: command not found`
> 
> I reverted the commit.

Sorry, I shouldn't review things late. I have proposed something that should work

File: io.github.endless_sky.endless_sky.json (hunk 1 above)
Comment (bilelmoussaoui) [inline io.github.endless_sky.endless_sky.json:39]:
```suggestion
        "CPPPATH=/app/include python3 /app/bin/scons PREFIX=/app install -j $FLATPAK_BUILDER_N_JOBS",
```
That's the change I wanted to suggest

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

File: io.github.endless_sky.endless_sky.json (hunk 1 above)
Comment (corecontingency) [reply io.github.endless_sky.endless_sky.json]: Thank you!

File: io.github.endless_sky.endless_sky.json (hunk 2 above)
Comment (bilelmoussaoui) [inline io.github.endless_sky.endless_sky.json:38]:
```suggestion
```

Comment (corecontingency) [reply io.github.endless_sky.endless_sky.json]: Whoops! Forgot that.

**Comment by corecontingency** [comment]
bot, build io.github.endless_sky.endless_sky

**Pull request closed without merging**